.github/workflows/validate-workflows.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate GitHub Actions Workflows
+
+on:
```

Comment:
These triggers feel like they were copy pasted from somewhere - is there any reason for them to be different than triggers in other workflows - that is, on push or pull request to master? Particularly the schedule trigger, since the build should be immutable, but the others too.

Reply:
addressed